packages/ses/src/lockdown.js

```diff
@@ -385,6 +386,19 @@ export const repairIntrinsics = (options = {}) => {
     // must be the operation that modifies the intrinsics.
     tamedHarden(intrinsics);
 
+    // Harden Symbol
+    tamedHarden(globalThis.Symbol);
+
+    // Harden evaluators
+    tamedHarden(globalThis.Function);
+    tamedHarden(globalThis.eval);
+    tamedHarden(globalThis.Compartment);
+
+    // Harden initialGlobalPropertyNames
+    for (const prop of getOwnPropertyNames(initialGlobalPropertyNames)) {
+      tamedHarden(globalThis[prop]);
+    }
```

Comment:
Nit: given the status of `Symbol` as effectively an `initialGlobalPropertyNames`, move its standalone hardening here

Reply:
38a92c087